src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -302,7 +300,9 @@ def _flexible_server_params(command_group):
                        help='Period of time (UTC) designated for maintenance. Examples: "Sun:23:30" to schedule on Sunday, 11:30pm UTC. To set back to default pass in "Disabled".')
             c.argument('tags', tags_type)
             if command_group == 'mysql':
-                c.argument('sku_name', options_list=['--sku-name'], validator=mysql_sku_name_validator,
+                c.argument('tier', options_list=['--tier'],
+                           help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized ')
```

Comment:
For allowed values. prefer using get_enume_type()

Reply:
Addressed in the SDK comment.

Reply:
I mean all allowed values for tier and sku_name. Could we also get the allowed array from sdk?

Reply:
Yes we do have validators for tier, sku_name, storage size validated by the array from sdk. We will include that for the next train.